### DISCUSSION TITLE
Problem in adding accounts - Solved!

### DISCUSSION THREAD

**Discussion by omidRMR** [masterking32/MasterCryptoFarmBot#16] [Q&A] [answered]
hi and thank you for creating such an amazing bot @masterking32  @Fy0urM  @tboy1337  @AmirhN1382 

I have problems with the account-adding procedure:
 
- by using API I get the following message (please note that I am using a free VPN and I've disabled my firewall): 
**"Unable to connect due to network issues: Connection timed out"**  

- I tried the second method and got the TgWebAppInitData, but the wiki page didn't mention what should be done after getting this data. Should we add an option in config.py? Apparently, "add_acount.py" does not support anything except telegram sessions.

**Comment by AmirhN1382** [comment]
Hey. Not sure about the first error, never received it.
About WebIinitData, you need to add them one-by-one for each bot ( I assume, I'm not sure how they work tbh, but I think it's better to get them for each bot and also maybe update by time to time? ). And as we don't have any modules/bot ready right now, you can't add them. Please wait until there are modules, so you can add your InitDatas there, and we will complete the Wiki section and guide you where to put InitDatas to.

**Reply by omidRMR** [reply]
Thank you

**Reply by omidRMR** [reply, marked as answer]
Problem solved. My VPN was not applied to all network activities. Now it is working! Thank you
